.github/workflows/main.yml

```diff
@@ -34,41 +27,20 @@ jobs:
           path: build/dist
           if-no-files-found: error
 
-      - name: Upload the build bypasses core file
-        uses: actions/upload-artifact@v2.2.4
-        with:
-          name: FastForward_chromium_bypasses.js
-          path: build/FastForward.chromium/injection_script.js
-          if-no-files-found: error
-
   Firefox:
     runs-on: ubuntu-latest
     
     steps:
-      - uses: actions/setup-node@v3.4.1
-        with:
-          node-version: 17
-      
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
       
-      - name: Install required node packages
-        run: npm install
-      
       - name: Make package
-        run: node scripts/build.js firefox
+        run: bash scripts/firefox.sh nover
```

Comment:
The build.js should be updated o include a fix, also the nicer should be appended to the run command to make sure it is not building the dev version